### PR TITLE
[TBCM-271-276] all npm audit fixes + lint fix

### PR DESCRIPTION
**Pull request by kushal-arora-fw**

### CODE REVIEW

**Review by dbayly-freshworks — COMMENTED** [review]
For the most part this looks ok, but I have some concerns about how many return types we are removing, and turning off no explicit any.

File: .eslintrc.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ module.exports = {
     ],
     'no-console': 'error',
     '@typescript-eslint/no-var-requires': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```
[the remark this answers is not in the document]
Comment (kushal-arora-fw) [reply .eslintrc.js:23]: Agree - isn't a best practice. But here's the background -  the new Typescript update started throwing error on existing code. I got two options, One is to fix ALL the explicit 'any's in the application, and the other is turning off. 

What are your thoughts around it?

[the remark this answers is not in the document]
Comment (dbayly-freshworks) [reply .eslintrc.js:23]: Ideally we should be having the correct types for everything. I think it makes sense to add an eslint ignore along with a TODO comment anytime we want to ignore 'no-explicit-any'. That way, in future, any "anys" will get flagged in the linting step.

[the remark this answers is not in the document]
Comment (kushal-arora-fw) [reply .eslintrc.js]: Updated as suggested. All explicit 'any's addressed. Either suppressed and the ones fixable - appropriately typed.

File: apps/api/package.json [hunk 2]
```diff
@@ -66,6 +66,7 @@
     "@types/aws-lambda": "8.10.84",
     "@types/express": "4.17.13",
     "@types/js-yaml": "4.0.5",
+    "@types/multer": "^1.4.11",
```
Comment (dbayly-freshworks) [inline apps/api/package.json:69]: Nit - We should lock this package version

**Review by dbayly-freshworks — APPROVED** [review]
This is a massive PR, but I think its ok. I have one nit.

**Pull request merged**